scripts/checkpatch.pl

```diff
@@ -3054,8 +3054,9 @@ sub process {
 				    $newindent ne $goodspaceindent) {
 
 					if (CHK("PARENTHESIS_ALIGNMENT",
-						"Alignment should match open parenthesis\n" . $hereprev) &&
+						"Alignment should match open parenthesis newindent = " . "$newindent" . "gt =  " . "$goodtabindent" . "gs = " . "$goodspaceindent" . "\n" . $hereprev) &&
```

Comment:
to add checkpatch fix in a separate patch (also on main branch if it fix an issue)